iotedgedev/envvars.py

```diff
@@ -142,8 +121,6 @@ def load(self, force=False):
                 "Variable that caused exception: {0}".format(str(ex))
                 raise ValueError(msg)
 
-        self.clean()
```

Comment:
>self.clean() [](start = 8, length = 12)

why removing the clean here?

Reply:
Correct me if I'm wrong, but the implementation above was for python 2 only, so it serves no purpose for python 3.